raiden/network/proxies/token_network.py

```diff
@@ -1444,20 +1445,38 @@ def close(
                 cannot be recovered.
         """
 
-        if signature != EMPTY_SIGNATURE:
-            canonical_identifier = CanonicalIdentifier(
-                chain_identifier=self.proxy.contract.functions.chain_id().call(),
-                token_network_address=self.address,
-                channel_identifier=channel_identifier,
-            )
+        canonical_identifier = CanonicalIdentifier(
+            chain_identifier=self.proxy.contract.functions.chain_id().call(),
+            token_network_address=self.address,
+            channel_identifier=channel_identifier,
+        )
+
+        our_signed_data = pack_balance_proof_update(
```

Comment:
The name `pack_balance_proof_update` sounds obsolete because it is now used not only for balanceProofUpdate calls, but also for closeChannels() call. Is  `pack_signed_balance_proof` a better name?

Reply:
Hmm you have a point. I think the name you propose sounds better. What do you think @hackaugusto ?

Reply:
good with me :)

Reply:
Then I do replace-string.